README.md

```diff
@@ -45,7 +45,11 @@ This stateless service demostrates a watch dog application which is monitoring t
 For compiling the samples use gradle. This should be configured as part of the Service Fabric Java SDK installation.
 
 ## Deploying the samples
-All the samples once compiled can be deployed immediately using the install.sh scripts provided along with the sample. These scripts underneath uses azurecli. Before running the scripts you need to first connect to the cluster using azurecli. 
+All the samples once compiled can be deployed immediately using the install.sh scripts located inside Scripts folder of every sample. These scripts underneath uses azurecli. Before running the scripts you need to first connect to the cluster using azurecli. 

```

Comment:
We need to capture the fact that the script must be executed from the root level. If someone goes inside the Scripts folder and then executes install.sh it is likely to fail